addons/l10n_cl/views/report_invoice.xml

```diff
@@ -201,8 +192,6 @@
 
     </template>
 
-
-    <!-- FIXME: Temp fix to allow fetching invoice_documemt in Studio Reports with localisation -->
```

Comment:
why do you remove this?

Reply:
restored in 914a6dd1a157c544976944af002c33cfae06946d